examples/Protobuf/proto/compile_linux.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+~/.nuget/packages/grpc.tools/1.16.0/tools/linux_x64/protoc \
```

Comment:
yeah. though it's installed by the project, so it's on us to get it right if we do change it. And probably a pretty good chance of that happening, since if we change the version, we'll be regenerating the classes which will remind us.